packages/design-system/rollup.config.js

```diff
@@ -0,0 +1,35 @@
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+import commonjs from "@rollup/plugin-commonjs";
+import typescript from "rollup-plugin-typescript2";
+import url from "@rollup/plugin-url";
+import svgr from "@svgr/rollup";
+import postcss from "rollup-plugin-postcss";
+import path from "path";
+import postcssImport from "postcss-import";
+import { terser } from "rollup-plugin-terser";
+
+export default {
+  input: ["src/index.ts", "src/Tooltip/index.tsx"],
```

Comment:
I think we should have a mechanism overhere that will include all the index files from each component. For example - 
`input: ['src/**/index.tsx']`

Reply:
Updated